lib/check/models.py

```diff
@@ -26,7 +26,30 @@ def load_check_metadata(metadata_file: str) -> dict:
         return check_metadata
 
 
-# Check all values
+class ComplianceItem(BaseModel):
+    Control: List[str]
+    Framework: str
+    Group: List[str]
+    Version: str
+
+
+class Code(BaseModel):
+    NativeIaC: str
+    Terraform: str
+    Cli: str
```

Comment:
I prefer `CLI`, what do you think?
```suggestion
    CLI: str
```